doc/import_external_docs.ps1

```diff
@@ -1,6 +1,7 @@
 param(
-  [Parameter(ValueFromPipeLineByPropertyName = $true)]
-  $branches = $null
+    [Parameter(ValueFromPipeLineByPropertyName = $true)]$branches = $null,
+    [Parameter(ValueFromPipeLineByPropertyName = $true)][string]$contributor_git_url = $null,
+    [Parameter(ValueFromPipeLineByPropertyName = $true)][string[]]$forks_to_import = $null
```

Comment:
Thanks for your contribution! Could you also update the docs here: https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/docfx.md ?